src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -354,7 +356,10 @@ def check_features_to_update(features_to_update):
             update_azure_rbac = True
         elif feature == "custom-locations":
             update_cl = True
-
+    if not update_cl:
+        if update_cluster_connect:
+            logger.warning("Cluster connect feature cant be disabled unless custom locations feature is also disabled.")
```

Comment:
Error out here is more consistent with what user expected to happen.

Reply:
This validation should not be here. Validate it when we check the helm values for custom locations. Added a comment for that.

Reply:
I'd asked in the group, we are not supposed to fail the onboarding if user passes some incorrect options.